app/config/galoy-instances.ts

```diff
@@ -1,42 +1,84 @@
 import { NativeModules } from "react-native"
 
+// this is used for local development
+// will typically return localhost
 const scriptHostname = (): string => {
   const { scriptURL } = NativeModules.SourceCode
   const scriptHostname = scriptURL?.split("://")[1].split(":")[0] ?? ""
   return scriptHostname
 }
 
-export const possibleGaloyInstanceNames = ["BBW", "Staging", "Local", "Custom"] as const
+export const standardInstances = ["Main", "Staging", "Local"] as const
+export const possibleGaloyInstanceNames = [...standardInstances, "Custom"] as const
 export type GaloyInstanceName = (typeof possibleGaloyInstanceNames)[number]
 
+export type StandardInstance = {
+  id: "Main" | "Staging" | "Local"
+}
+
+export type CustomInstance = {
+  id: "Custom"
+  name: string
+  graphqlUri: string
+  graphqlWsUri: string
+  posUrl: string
+  lnAddressHostname: string
+  blockExplorer: string
+}
+
+export type GaloyInstanceInput = StandardInstance | CustomInstance
+
 export type GaloyInstance = {
-  name: GaloyInstanceName
+  id: GaloyInstanceName
+  name: string
   graphqlUri: string
   graphqlWsUri: string
   posUrl: string
   lnAddressHostname: string
+  blockExplorer: string
+}
+
+export const maybeAddDefault = (input: GaloyInstanceInput): GaloyInstance => {
```

Comment:
This name doesnt really make sense to me. Maybe something like `resolveGaloyInstanceOrDefault` could be better.